examples/workflow-glsp/src/model.ts

```diff
@@ -58,6 +59,13 @@ export class TaskNode extends RectangularNode implements Nameable, WithEditableL
     }
 }
 
+export class MarqueeNode extends RectangularNode {
+    static readonly DEFAULT_FEATURES = [connectableFeature, deletableFeature, selectFeature, boundsFeature,
```

Comment:
I think we should remove all unneeded features here to avoid potential unintended side effects.
IMO it should be enough to only activate the `boundsFeature`